NeoML/docs/en/API/NN/TiedEmbeddingsLayer.md

```diff
@@ -0,0 +1,42 @@
+# CTiedEmbeddingsLayerd Class
+
+<!-- TOC -->
+
+- [CTiedEmbeddingsLayerd Class](#ctiedembeddingslayerd-class)
+  - [Settings](#settings)
+    - [EmbeddingsLayerName](#embeddingslayername)
+    - [ChannelIndex](#channelindex)
+  - [Inputs](#inputs)
+  - [Outputs](#outputs)
+
+<!-- /TOC -->
+
+Class implements tied embeddings layer.
+
+## Settings
+
+### EmbeddingsLayerName
+
+```c++
+void SetEmbeddingsLayerName( const char* name )
+```
+Embeddings layer `name`.
+
+
+### ChannelIndex
+```c++
+void SetChannelIndex( int val );
+```
+Embedding channel index `val`
+
+## Inputs
+
+Each input accepts a blob with `BatchLength * BatchWidth * ListSize` objects with size `1 * 1 * 1 * EmbeddingsSize`.
+
+`EmbeddingsSize` - size of the embeddings vector.
+
+## Outputs
+
+Each output contains a blob with `BatchLength * BatchWidth * ListSize` objects with size  `1 * 1 * 1 * EmbeddingsCount`.

```

Comment:
Clarify number of outputs and which dimensions match with corresponding input and which don't. [Eng example](https://github.com/neoml-lib/neoml/blob/05168e5233d7dd9ba40acdec4a0095dc05cce96c/NeoML/docs/en/API/NN/DiscreteFeaturesLayers/MultiChannelLookupLayer.md)